src/main/scala/fairysf/format/Uci.scala

```diff
@@ -75,14 +82,17 @@ object Uci {
         promotion = Role.promotable(gf, promS)
       } yield Move(orig, dest, promotion)
 
-    val moveR = s"^${Pos.posR}${Pos.posR}(\\+?)$$".r
-    val moveP = s"^${Pos.posR}${Pos.posR}${Role.roleRr}$$".r
+    val moveR  = s"^${Pos.posR}${Pos.posR}(\\+?)$$".r
+    val moveP  = s"^${Pos.posR}${Pos.posR}${Role.roleRr}$$".r
     val movePR = s"^${Pos.posR}${Pos.posR}${Role.rolePR}$$".r
   }
 
   case class Drop(role: Role, pos: Pos) extends Uci {
 
-    def uci = s"${role.pgn}@${pos.key}"
+    def lilaUci = s"${role.pgn}@${pos.key}"
+    def fairySfUci = s"${role.pgn}@${pos.key}"
+    def fishnetUci     = fairySfUci // Use the fairySfUci
```

Comment:
Is this to document some ordering of the uci flow, as they are all the same, why not 
def fairySfUci = lilaUci

Reply:
Done.